compiler/src/dotty/tools/dotc/transform/Splicing.scala

```diff
@@ -208,6 +208,14 @@ class Splicing extends MacroTransform:
 
     override def transform(tree: tpd.Tree)(using Context): tpd.Tree =
       tree match
+        case tree: Select if tree.isTerm && isCaptured(tree.symbol) =>
+          tree.symbol.allOverriddenSymbols.find(sym => !isCaptured(sym.owner)) match
```

Comment:
Does it matter if there are multiple overridden symbols that are not captured? I think it doesn't but just want to make sure.

Reply:
It should no matter.